app/uk/gov/hmrc/bindingtariffclassification/repository/SearchMapper.scala

```diff
@@ -102,6 +114,41 @@ class SearchMapper {
     "$options" -> "i"
   }
 
+  private def filteringByStatus(search: Set[PseudoCaseStatus.PseudoCaseStatus]): (String, JsValue) = {
```

Comment:
PseudoCaseStatus.PseudoCaseStatus looks weird

Reply:
If I pull it out into its own file I think that should help

Reply:
Fixed